src/coordinate.py

```diff
@@ -1,15 +1,43 @@
 """Coordinate module"""
 from __future__ import annotations
-from typing import Tuple
+from typing import Tuple, Optional, Any
 
 
 class Coordinate(object):
     """Abstract each board position."""
 
-    def __init__(self, coordinate: Tuple[int, int]) -> None:
+    def __init__(self, coordinate: Any, secondCoord: Optional[int]) -> None:
```

Comment:
If we default the second parameter to `None` it will make it much cleaner to call `Coordinate()`.

- [ ] Set `secondCoord` to default to `None`
- [ ] Remove `None` from `Coordinate()` calls